src/Middleware/SpaServices.Extensions/src/SpaApplicationBuilderExtensions.cs

```diff
@@ -35,7 +35,7 @@ public static void UseSpa(this IApplicationBuilder app, Action<ISpaBuilder> conf
 
             // Use the options configured in DI (or blank if none was configured). We have to clone it
             // otherwise if you have multiple UseSpa calls, their configurations would interfere with one another.
-            var optionsProvider = app.ApplicationServices.GetService<IOptions<SpaOptions>>();
+            var optionsProvider = app.ApplicationServices.GetService<IOptions<SpaOptions>>()!;
```

Comment:
Does GetRequiredService not work here?

Reply:
Yeah. It will never be null so didn't seem worth it to supress the warning.